src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -229,15 +233,15 @@ def forward(
 
             q_rot = query[:, :, :, : self.rotary_dim]
             q_pass = query[:, :, :, self.rotary_dim :]
-
-            sincos = fixed_pos_embedding(k_rot, 1, seq_len=seq_len)
+            
+            sincos = (self.sin, self.cos)
             k_rot = apply_rotary_pos_emb(k_rot, sincos, offset=offset)
             q_rot = apply_rotary_pos_emb(q_rot, sincos, offset=offset)
 
             key = torch.cat([k_rot, k_pass], dim=-1)
             query = torch.cat([q_rot, q_pass], dim=-1)
         else:
-            sincos = fixed_pos_embedding(key, 1, seq_len=seq_len)
+            sincos = (self.sin, self.cos)
```

Comment:
This is what I mentioned above. This block is for `if self.rotary_dim is None`, and currently, this PR hasn't address this case in `__init__`